test/src/e2e_vm_tests/test_programs/should_pass/language/break_and_continue/src/main.sw

```diff
@@ -0,0 +1,70 @@
+script;
+
+const J = 10;
+const K = 20;
+const I = 30;
+const N = 100;
+
+fn break_test() -> u64 {
+    let m = 5;
```

Comment:
I don't understand the point of all of these variables and calculations. Could this be made simpler? Why is the output so large?

Reply:
Good question. I really tried to go wild here to stress test `break` and `continue` in a complex setting with many nested loops. I can probably simplify the math a bit but I was worried about losing cover my mistake by choosing the wrong numbers. Crafting a better test is certainly possible here though. I can do that tomorrow.

Reply:
I like the complexity but I don't want to be the one who has to debug it if it starts failing. :smile:

Reply:
I added two smaller tests and reduced the complexity of the existing test a bit. I do want to keep the nested structure and sibling loops and combine break and continue in one test to see what happens.